File: Solutions/JavaScript/pattern011.js
// Pattern Programs - 011 [JavaScript]
// Github Repository: https://github.com/mxsena/pattern-programs
let pattern = ""
for(let i=1; i<=5; i++){
    for(let j=1; j<=3; j++){
        pattern += i+ " "+j+" "
    }
    pattern += "\n"
}
console.log(pattern)

File: Solutions/C/pattern011.c
// Pattern Programs - 011 [C]
// Github Repository: https://github.com/mxsena/pattern-programs
#include <stdio.h>

int main(void)
{
    int i, j;
    
    for (i = 1; i <= 5; i++)
    {
        for (j = 1; j <= 3; j++)
        {
            printf("%d %d ", i, j);
        }
        printf("\n");
    }
    return 0;
}